src/Core/Geometry/Normal/Normal.cpp

```diff
@@ -35,12 +35,50 @@ void uniformNormal( const VectorArray< Vector3 >& p, const VectorArray< Triangle
         normal[k] += triN;
 //}
     }
+    #pragma omp parallel for
+    for( uint i = 0; i < N; ++i ) {
+        if( !normal[i].isApprox( Vector3::Zero() ) ) {
+            normal[i].normalize();
+        }
+    }
+}
+
+
+void uniformNormal( const VectorArray< Vector3 >& p, const VectorArray< Triangle >& T,
+                    const std::vector<uint>& duplicateTable, VectorArray< Vector3 >& normal ) {
+    const uint N = p.size();
+    normal.clear();
+    normal.resize( N, Vector3::Zero() );
 //#pragma omp parallel for
```

Comment:
Why is this commented out ?

Reply:
I guess we can get rid of it together with the `#pragma omp critical` comments, because it doesn't seem to be faster.
( same for within the above method )

Reply:
Ok, so parallelization doesn't improve performances here.